.storybook/main.js

```diff
@@ -17,7 +17,7 @@ module.exports = {
 		'@storybook/addon-a11y/register',
 		'@storybook/addon-knobs/register',
 	],
-	webpackFinal: config => {
+	webpackFinal: (config) => {
```

Comment:
😱 Oh no 💔! I loved it when didn't have brackets for those cases.

Reply:
I know... I also liked not having it but I guess they thought it was more useful to put them mandatory. On the brightside, you should be able to write them without and then auto-format it to having them

Reply:
Haha yeah, it's a good way to ignore 😅!

Reply:
FWIW, they had good reasons for [changing the default](https://prettier.io/blog/2020/03/21/2.0.0.html):

> At first glance, avoiding parentheses in the isolated example above may look like a better choice because it results in less visual noise. However, when Prettier removes parentheses, it becomes harder to add type annotations, extra arguments, default values, or a variety of other things. Consistent use of parentheses provides a better developer experience when editing real codebases, which justifies the change.

(You could override the setting, but in liferay-portal and the other repos we're going to accept the new default.)